spec/jobs/voice_otp_sender_job_spec.rb

```diff
@@ -14,14 +14,10 @@
 
       calls = FakeVoiceCall.calls
 
-      code = '1234'.scan(/\d/).join(', ')
-      message = t('jobs.voice_otp_sender_job.message', code: code)
-      url_message = URI.escape(message)
-
       expect(calls.size).to eq(1)
       call = calls.first
       expect(call.to).to eq('555-5555')
-      expect(call.url).to include(url_message)
+      expect(call.url).to include('code=1234')
```

Comment:
we are still sending the same message, shouldn't we still check for it in our specs?